tests/attr/test_class_summarizer.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+import torch
+
+from captum.attr import ClassSummarizer, CommonSummarizer
+
+from .helpers.utils import BaseTest
+
+
+class Test(BaseTest):
+    def class_test(self, data, classes, x_sizes):
+        summarizer = CommonSummarizer(ClassSummarizer)
```

Comment:
I think it would be nice to have something like summarizer = ClassSummarizer(...)